hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/transfer/AdjustFungibleTokenChangesStepTest.java

```diff
@@ -146,7 +152,7 @@ void doesTokenBalanceChangesWithAllowances() {
         assertThat(receiverAccountBefore.numberPositiveBalances()).isEqualTo(2);
         assertThat(senderRelBefore.balance()).isEqualTo(1000L);
         // There is an association happening during the transfer for auto creation
-        assertThat(receiverRelBefore.balance()).isEqualTo(1);
+        assertThat(receiverRelBefore.balance()).isEqualTo(0);
```

Comment:
Why wasn't this assert needed (or useful) also at L106 above? (Only instance in these tests modified in this PR where it wasn't.)